### ISSUE TITLE
This skill has reached the usage limit for today.

### ISSUE THREAD

**Issue by Ambro86** [ndg63276/alexa-youtube#134]
Hi, for a few hours using this skill gives me the message in question and I can no longer play any videos from youtube. Why? I specify that I only use this skill and that it is not possible that it has reached the limit of use. Is It possible that youtube has somehow blocked the skill?

**Comment by Ambro86** [comment]
Reading another user who had a similar problem, I created a new youtube key developer. Now it works again. The question is: I created the previous youtube key 4 days ago. Will a new one be created every 4 days? Because if so it means that it is youtube that blocks everything. Therefore it would not be worth it.

**Comment by ndg63276** [comment]
As I understand it, creating a new dev key under the same account won't help, as the Google API limits are per account. You can of course make a new Google account.

But no, you shouldn't have to make a new key every few days. I have one key that I use for about 50 users. However, Google has gradually increased the quota for it over time, so maybe you have just asked too much in a short space of time.

If you go to https://console.cloud.google.com/iam-admin/quotas, you should be able to request a higher quota.

**Comment by Ambro86** [comment]
Hello, thanks for your reply. I visited the link you gave me, but if I understand correctly the values of the limits are already at the maximum.
YouTube Data API v3

Queries per day 
9
10.000
Queries per 100 seconds per user 
0,021
300.000
Queries per 100 seconds
3.000.000
In fact, if I try to increase the values, it doesn't let me do it, because it tells me to enter a value between 0 and 10000, for example, for queries per day.
If I understand correctly I used youtube far below the limit. Of the 10,000 queries only 9 have been used.
What should I do if this block happens again?
